loops-and-arrays.js

```diff
@@ -41,6 +55,11 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+console.log(squares);
+
+for (let i=0; i <squares.length; i++) {
+    console.log("Het volume van " + squares[i] + " is " + squares[i]*squares[i]*squares[i]);
```

Comment:
Op zich prima, maar je had ook `squares[i]ˆ3`kunnen doen. In principe kost het wat CPU tijd om gegeven een index een waarde uit een array te halen. Dus drie keer dezelfde waarde ophalen is dan zonde.